book/website/community-handbook/governance/gov-constitutional.md

```diff
@@ -0,0 +1,131 @@
+
+The Chair of the Steering Council holds their role for a calendar year, starting in January and handing over their responsibilities in December.
+
+The Chair joins as Chair Elect for one year, holds the role of Chair for one year, and remains on the Steering Council for one year as Past Chair.
+
+In order to be elected the Chair must be an existing chair of a working group or delivery group.
+However, if they step down from those responsibilities before the end of their three year commitment to The Turing Way (as Chair Elect, Chair and Past Chair) they may remain in post.
+
+For example, a project associated with a delivery group could end while the Chair of the delivery group is serving as chair of the Steering Committee.
+This should not mean that the Chair of the Steering Committee is required to end their service to *The Turing Way* community at this consitutional level.
+However, it is always possible for a Chair to step down if they choose to.
+
+If the Chair Elect steps down from their role the Steering Committee will elect another member into the role.
+
+If the Chair steps down from their role the Chair Elect will be asked to step into the role of Chair and will serve for the remainder of the Chair's term as well as their full year as Chair.
+
+If the Chair Elect does not want to take on the Chair role earlier than planned then the Steering Committee will elect a Chair from their membership. 
+
+The role of Chair can be shared by up to three Co-Chairs as described in the <!-- MAKE INTO LINK-->role of a chair page in this chapter.
+
+## Ways of working
+
+### Synchronous collaboration
+
+The Steering Council will meet every 3 months online.
+
+They will aim to host their meetings around 1 month before the community forum events to maintain a regular cadance of communication between the Steering Council and the broader community.
+
+It is up to the Steering Council to decide how long each meeting should last.
+
+It is the responsibility of the Chair to set the agenda for these meetings.
+They may choose to - for example - focus on particularly thorny topics for discussion, or to invite presentations on activity across the project to keep all Steering Council members informed.
+
+The Steering Council may also hold ad hoc meetings to discuss specific topics in order to facilitate empowered, transparent and inclusive decision making across the project. 
+
+### Asynchronous collaboration
+
+Steering Council members will be added to a private slack channel in *The Turing Way* slack workspace.
+They can use this space to share knowledge and to ask for input for ongoing discussions.
+
+All requests for decisions by the Steering Council are documented as issues in the governance repository in *The Turing Way* GitHub organisation.
+Steering Council members are expected to monitor these issues and contribute on topics related to the interests and needs of their working group or delivery group members.
```

Comment:
I like this focus on the "official" work being done in public rather than on Slack.